### PR TITLE
[ENH, MRG] Add manual alignment procedure to ieeg localize tutorial

### PR DESCRIPTION
**Pull request by alexrockhill**
Here is the procedure for doing the alignment in freeview as the `Dipy` alignment doesn't always work.

cc @leosunpsy (feel free to review)

### CODE REVIEW

File: tutorials/clinical/10_ieeg_localize.py [hunk 1]
```diff
@@ -192,27 +192,28 @@ def plot_overlay(image, compare, title, thresh=None):
...
-#           CT_aligned_pre = nib.load(op.join(misc_path, 'seeg',
-#                                             'sample_seeg_CT_aligned.mgz'))
-#           CT_aligned = resample(moving=np.asarray(CT_aligned_pre.dataobj),
-#                                 static=np.asarray(T1.dataobj),
-#                                 moving_affine=CT_aligned_pre.affine,
-#                                 static_affine=T1.affine)
-#
-#    The rest of the tutorial can then be completed using ``CT_aligned``
-#    from this point on.
+# .. note::
```
Comment (larsoner) [inline tutorials/clinical/10_ieeg_localize.py:195]: Have you tried locally `.. sidebar::` instead? You can do for example:
```
PATTERN=10_ieeg make html_dev-pattern
```
and you should be able to look at the output to see if it's better or worse

Comment (larsoner) [reply tutorials/clinical/10_ieeg_localize.py:195]: ... though I guess this is an entire section. It's a bit weird to put an entire section in a note, no?

Comment (alexrockhill) [reply tutorials/clinical/10_ieeg_localize.py:195]: It actually doesn't need to be executed and it doesn't look good, it ends up looking out of place

**Comment by alexrockhill** [comment]
Ok, this looked good to me locally

**Comment by larsoner** [comment]
Thanks @alexrockhill

**Comment by leosunpsy** [comment]
Thanks @alexrockhill

**Pull request merged**